### ISSUE TITLE
Tooltips no longer display since PR #834

### ISSUE THREAD

**Issue by falquaddoomi** [BRCAChallenge/brca-exchange#837]
PR #813 introduced hover-over tooltips; the content for those tooltips is scraped from the help markdown with some assumptions about where the help markdown is stored and how field help entries are formatted.

PR #834 introduced a new structure for the help documentation, including both content changes and changes to where the markdown is stored in the filesystem. The parsing code for extracting tooltips (predictably) hadn't been updated to reflect these changes, so the tooltips and help links ceased to display.

**Comment by zfisch** [comment]
has this already been addressed?

**Comment by falquaddoomi** [comment]
@zfisch Yes, the main issue was addressed in PR #838 and pushed to beta. I thought I'd fixed the issue with the help sections not opening in PR #839, but apparently it's still broken...I'll reopen the PR and see if I can fix it.

FYI, the problem I *thought* I'd fixed was caused by minification settings for HTML generated from markdown being different in production than they are in dev. Specifically, the minifier was removing quotes around attributes that didn't contain whitespace (e.g. id="xyz"), and the code that checked to see if a section should be open based on the fragment was looking for those quotes. I relaxed the check to also matching non-quoted id's, but apparently that wasn't enough.
